Algorithm/QCAlgorithm.Indicators.cs

```diff
@@ -1332,12 +1332,17 @@ public SubscriptionDataConfig GetSubscription(Symbol symbol, TickType? tickType)
             SubscriptionDataConfig subscription;
             try
             {
+                // deterministic ordering is required here
+                var subscriptions = SubscriptionManager.Subscriptions
+                    .OrderBy(x => x.Symbol)
+                    .ThenBy(x => x.TickType);
```

Comment:
since we're filtering by `symbol` below, I don't think we need to perform an order by based on symbol. Please test removing the order by symbol to see if it still works as expected

Reply:
You're right, order by symbol is not needed. Updated PR